docs/using.rst

```diff
@@ -188,6 +190,35 @@ expected output is::
 .. _README: ../components/centos/centos-k8s-singlenode-setup/README.rst
 .. _Vagrantfile: ../components/centos/centos-k8s-singlenode-setup/Vagrantfile
 
+Vagrant bi-directional folder sync
+==================================
+
+For basic usage please refer `Vagrant documentation. <https://www.vagrantup.com/docs/synced-folders/basic_usage.html>`_
+
+Vagrant bi-directional sync folder is a very useful feature in Vagrant as it is the simplest way to move files (e.g. code) between
+the workstation and the Vagrant guest.
+
+Here are options for bi-directional sync works out of box with ADB Vagrant box.
+
+* `vagrant-sshfs <https://github.com/dustymabe/vagrant-sshfs>`_ : It works with Linux/GNU, OS X and Microsoft Windows.
+* `NFS <https://www.vagrantup.com/docs/synced-folders/nfs.html>`_ : For Linux/GNU and OS X.
+
+
+These solutions should work for both Virtualbox and Libvirt/KVM providers. 
+
+However there are some other alternatives which can be used with ADB. Theoretically these should work with ADB but at this point of time these are not properly tested with ADB.
```

Comment:
+There are also some other alternatives which, however,  are currently not properly tested with ADB: